firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/AppDistributionRelease.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.appdistribution;
+
+import androidx.annotation.NonNull;
+
+/**
+ * Interface for AppDistributonRelease object returned by checkForUpdate() and
+ * updateToLatestRelease()
+ */
+public interface AppDistributionRelease {
```

Comment:
This looks like a POJO to me, why do we need this to be an interface rather than a `public final class`? Do we anticipate different implementations with non-trivial logic as opposed to just treating this class as a "bag" of values?

Reply:
Done, this should be a POJO